packages/platforms/accton/x86-64/x86-64-accton-as7326-56x/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,175 @@
+#include <unistd.h>
+#include <fcntl.h>
+#include "platform_lib.h"
+#include <onlp/platformi/sfpi.h>
+#include "x86_64_accton_as7326_56x_log.h"
+
+
+static int _onlp_file_write(char *filename, char *buffer, int buf_size, int data_len)
```

Comment:
Can you use the existing onlp_file_t* APIs instead of rewriting your own?
Please see these available functions:
https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/base/any/onlp/src/onlplib/module/inc/onlplib/file.h